doc/DEMO_TESTNET.md

```diff
@@ -0,0 +1,327 @@
+
+We'll use testnet: no need to have a synced node!
+
+### Rust
+
+`revaultd`, `cosignerd` and `coordinatord` are written in Rust. Since we don't publish compiled binaries (yet!), 
+you'll need to have the Rust toolchain installed to be able to compile the projects.
+
+Please refer to https://www.rust-lang.org/tools/install for instructions on how to install Rust.
+
+Add `export PATH="$PATH:$HOME/.cargo/bin"` to your `.bashrc` or
+`.zshrc`.
+
+Double check that your rust toolchain is **at least** 1.53.0:
+
+```
+cargo --version
+```
+
+### Some text editor
+
+There are various keys that you'll need to keep during the whole configuration. 
+Obviously a text editor is not a good choice for storing real keys, but
+hey, we're in testnet. Open your favorite note-taking app.
+
+### The Revault ceremony
+
+The Revault setup requires to generate secrets for each participants in a safely way.
+For this testnet setup, the security is not the primary goal, so the
+process of generating and sharing keys is left as an exercise to the readers.
+
+The required secrets are
+
+**if you are a stakeholder**:
+
+- one bip32 xpriv and its xpub.
+- one noise private key and its public key
+
+**if you are a manager**:
+
+- one bip32 xpriv and its xpub.
+- one noise private key and its public key
+
+**if you are a stakeholder and a manager**
+
+- two bip32 xpriv and their xpubs (one key pair called the stakeholder
+  keys and the other one the manager keys)
+- one noise private key and its public key
+- one cosigner bitcoin private key with its public key
+- one cosigner noise private key and its public key
+
+You can generate bip32 key pair on bip32.org
+For a noise key pair and a cosigner bitcoin key pair you can do:
+
+`python3 -c 'import os;print(os.urandom(32).hex())'`
+
+### Docker (kinda optional)
+
+We need docker to spin up the `coordinatord`'s Postgre database.
+You can avoid using docker though, and just spin up a Postgre database yourself.
+
+## Downloading the repositories
+
+Let's start by creating a dedicated folder - this way if you don't like Revault it will be easier to erase everything :D
+
+```
+mkdir revault_tutorial
+cd revault_tutorial
+```
+
+### If you are a stakeholder:
+
+Download all the needed repositories: `cosignerd`, `revaultd`, `revault-gui`:
+```
+~/revault_tutorial$
+git clone -b 0.2 https://github.com/revault/cosignerd
+git clone -b 0.2 https://github.com/revault/revaultd
+git clone -b 0.2 https://github.com/revault/revault-gui
+```
+
+Then in each of the repositories:
+
+```
+cargo install --path .
+```
+
+Binaries will be installed in `~/.cargo/bin`.
+
+### If you are a manager:
+
+Download all the needed repositories: `revaultd`, `revault-gui`:
+```
+~/revault_tutorial$
+git clone -b 0.2 https://github.com/revault/revaultd
+git clone -b 0.2 https://github.com/revault/revault-gui
+```
+
+Then in each of the repositories:
+
+```
+cargo build
```

Comment:
Why do we require the stakeholder to install (L125) and the manager only to build?